release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -87,7 +87,9 @@ You can deploy an {product-title} cluster to multiple vSphere datacenters or reg
 
 [IMPORTANT]
 ====
-The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this feature requires the Container Storage Interface (CSI) storage class. A cluster that was upgraded from a previous release does not include the required CSI storage class.
+The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this type of cluster includes the Container Storage Interface (CSI) automatic migration feature. The CSI automatic migration feature sets CSI storage class as the default storage class. 
+
+If you want to upgrade a cluster from a previous release, you must manually enable the CSI automatic migration feature, and then ensure that CSI storage is set as the default storage class for your cluster.
```

Comment:
The VMware vSphere region and zone enablement feature is only available with a newly installed cluster,  because this requires vSphere CSI driver as default storage driver in the cluster. 

A cluster that was upgraded from previous release defaults to intree vSphere driver and hence CSI automatic migration must be enabled on such clusters before multiple zones and regions can be configured in upgraded clusters.